app/storybook/documentation/ui-kit/QueryHooks.mdx

```diff
@@ -0,0 +1,60 @@
+import { ArgTypes, Meta } from '@storybook/blocks'
+import LinkTo from '@storybook/addon-links/react'
+import { SourceCode } from '../../.storybook/blocks/SourceCode'
+import * as useCounterQueryStories from '../../../../packages/ui-kit/src/hooks/useCounter/useCounter.stories'
+
+import * as useLeaderboardStories from '../../../../packages/ui-kit/src/hooks/useLeaderboard/useLeaderboard.stories'
+
+import * as useTimeSeriesStories from '../../../../packages/ui-kit/src/hooks/useTimeSeries/useTimeSeries.stories'
+
+<Meta title="Customization/Query Hooks" />
+
+# Query Hooks
+
+Propel’s UI Kit library provides the following React components: Counter, TimeSeries, and Leaderboard. To query the GraphQL API, we have developed 
+Query Hooks including "useCounter", "useLeaderboard" and "useTimeSeries.
+
+## useCounter
```

Comment:
Should hook names be in backticks, like we do for component names in the other headings?
```suggestion
## `useCounter`
```

Reply:
I do not think so we should wrap it in backticks.